includes/class-wc-gateway-swedbank-pay-swish.php

```diff
@@ -63,12 +64,15 @@ public function __construct() {
 
 		$this->id           = 'payex_psp_swish';
 		$this->has_fields   = true;
-		$this->method_title = __( 'Swish', WC_Swedbank_Pay::TEXT_DOMAIN );
-		$this->icon         = apply_filters( 'wc_swedbank_pay_swish_icon', plugins_url( '/assets/images/swish.png', dirname( __FILE__ ) ) );
-		$this->supports     = [
+		$this->method_title = __( 'Swish', 'swedbank-pay-woocommerce-payments' );
+		$this->icon         = apply_filters(
+			'wc_swedbank_pay_swish_icon',
+			plugins_url( '/assets/images/swish.png', dirname( __FILE__ ) )
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function dirname() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)